tasks/configure_namespace.yml

```diff
@@ -92,4 +109,46 @@
   - pipeline-task-definitions
 
 
+# TODO: Cluster role: Admin does not have permissions to create/adjust limitranges!
+- name: Create clean up cronjobs
+  k8s:
+    state: present
+    definition:
+      apiVersion: batch/v1
+      kind: Job
+      metadata:
+        name: "{{ item.name }}"
+        namespace: "{{ ocp_namespace }}"
+      spec:
+        backoffLimit: 6
+        completions: 1
+        parallelism: 1
+        template:
+          spec:
+            containers:
+              - command:
+                  - /bin/bash
+                  - '-c'
+                  - >
+                  - "{{ item.template }}"
+                image: "{{ cronjob_image }}"
+                imagePullPolicy: IfNotPresent
+                name: build-pruner
+                resources: {}
+                terminationMessagePath: /dev/termination-log
+                terminationMessagePolicy: File
+            dnsPolicy: ClusterFirst
+            restartPolicy: OnFailure
+            schedulerName: default-scheduler
+            securityContext: {}
+            # serviceAccount: pruner
+            # serviceAccountName: pruner
+            terminationGracePeriodSeconds: 30
+  with_items:
+    - { name: "pod-pruner", template: "{{ lookup('template', 'templates/pod_pruner.sh') }}"}
+  tags:
+  - cronjobs
+  - source-containers
```

Comment:
Does the `source-containers` tag still have a purpose? If I'm not mistaken, this and the variables check are the only tasks that have it

Reply:
answered above. tag not used